packages/lesswrong/server/vulcan-lib/apollo-ssr/pageCache.ts

```diff
@@ -292,3 +292,16 @@ export function checkForMemoryLeaks() {
     console.log(`Possible memory leak: pageCacheContentsBytes=${pageCacheContentsBytes}`);
   }
 }
+
+export function printInFlightRequests() {
+  let inProgressRenderKeys: string[] = [];
+  for (let cacheKey of Object.keys(inProgressRenders)) {
+    for (let render of inProgressRenders[cacheKey]) {
```

Comment:
Minor, but I think `render.cacheKey == cacheKey` so this could be replaced with `inProgressRenderKeys = Object.keys(inProgressRenders)`.

If I'm wrong about that,  `inProgressRenderKeys = Object.values(inProgressRenders).flat()`would seem preferable to the nested for loops in my opinion.